lib/elastic_apm/span.rb

```diff
@@ -50,7 +50,7 @@ def initialize(
       context: nil,
       stacktrace_builder: nil,
       sync: nil,
-      sample_rate: nil
+      _sample_rate: nil
```

Comment:
If we don't use this, should we drop it?

Reply:
sure🤷🏻‍♀️